nixtract/find-attribute-paths.nix

```
@@ -16,11 +17,16 @@ let
 
   # Arguments have to be taken from environment when using `nix` command
   targetFlakeRef = builtins.getEnv "TARGET_FLAKE_REF";
+  targetAttribute = builtins.getEnv "TARGET_ATTRIBUTE_PATH";
   targetSystem = builtins.getEnv "TARGET_SYSTEM";
 
   # Get pkgs
   targetFlake = builtins.getFlake targetFlakeRef;
   targetFlakePkgs = lib.getFlakePkgs targetFlake targetSystem;
+  targetPackage =
```

I'm not convinced we could call it a "package".
I would rather see it as the "root" value from which we recurse.

```suggestion
  targetRootValue =
```